src/utils/openai.ts

```diff
@@ -4,14 +4,15 @@ const sanitizeMessage = (message: string) => message.trim().replace(/[\n\r]/g, '
 
 const deduplicateMessages = (array: string[]) => Array.from(new Set(array));
 
-const promptTemplate = 'Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff without prefacing it with anything:';
+const getPrompt = (lang: string, diff: string) => `Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff without prefacing it with anything, the response must be in the lang ${lang}:\n${diff}`;
```

Comment:
Is there a reason for using "lang" instead of "language"?

Reply:
Nope, no special reason for that, just an abbreviation, but changed it for readability.